setup.py

```diff
@@ -190,13 +190,12 @@ def get_git_sha() -> str:
         "vertica": ["sqlalchemy-vertica-python>=0.5.9, < 0.6"],
         "netezza": ["nzalchemy>=11.0.2"],
     },
-    python_requires="~=3.9",
```

Comment:
I would recommend leaving explicit support for 3.9 for a while longer, as we've only just deprecated 3.8.

Reply:
If we want to support Py3.9 then we don't need this PR